documentation/modules/creating-migration-quick-plan-existing-source.adoc

```diff
@@ -22,7 +24,31 @@ The *Create migration plan* pane opens. It displays the source provider's name a
+{project-short} validates the migration plan, and the *Plan details* page opens, indicating whether the plan is ready for use or contains an error. In all cases, the details of the plan are listed, and you can edit the items you filled in on the previous page. If you make any changes, {project-short} validates the plan again.
+
+. VMware source providers only (All optional):
+
+* *Preserving static IPs of Windows VMs*: By default, Virtual Network Interface Cards (VNICs) change during the migration process. This results in Windows VMs that are set with a static IP in vSphere losing their IP. To avoid this, click the Edit icon next to *Preserve static IPs*, and toggle the *Whether to preserve the static IPs* switch in the window that opens. Then click *Save*.
++
+{project-short} then issues a warning message about any VMs with a Windows operating system for which VNIC properties are missing. To retrieve any missing VNIC properties, run those VMs in vSphere in order for the VNIC properties to be reported to {project-short}.
+
+* *Entering a list of decryption passphrases for disks encrypted using Linux Unified Key Setup (LUKS)*: To enter a list of decryption passphrases for LUKS-encrypted devices, in the *Settings* section, click the Edit icon next to *Disk decryption passphrases*, enter the passphrases, and then click *Save*. You do not need to enter the passphrases in a specific order - for each LUKS-encrypted device, {project-short} tries each passphrase until one unlocks the device.
+
+* *Specifying a root device*: Applies to multi-boot VM migrations only. By default, {project-short} uses the first bootable device detected as the root device.
++
+To specify a different root device, in the *Settings* section, click the Edit icon next to *Root device* and choose a device from the list of commonly-used options, or enter a device in the text box.
++
+{project-short} uses the following format for disk location: `/dev/sd<disk_identifier><disk_partition>`. For example, if the second disk is the root device and the operating system is on the disk's second partition, the format would be: `/dev/sdb2`. After you enter the boot device, click *Save*.
++
+If the conversion fails because the boot device provided is incorrect, it is possible to get the correct information by looking at the conversion pod logs.
+
+. {rhv-short} source providers only (Optional):
+
+* *Preserving the CPU model of VMs that are migrated from {rhv-short}*: Generally, the CPU model (type) for {rhv-short} VMs is set at the cluster level, but it can be set at the VM level, which is called a custom CPU model.
+By default, {project-short} sets the CPU model on the destination cluster as follows: {project-short} preserves custom CPU settings for VMs that have them, but, for VMs without custom CPU settings, {project-short} sets their CPU model to the default CPU model in {virt}.
```

Comment:
that's better but we can make it a bit more technically accurate by:
```suggestion
By default, {project-short} sets the CPU model on the destination cluster as follows: {project-short} preserves custom CPU settings for VMs that have them, but, for VMs without custom CPU settings, {project-short} does not set a CPU model and thus, it is later set by {virt}.
```

Reply:
@ahadas Small tweak -- please check for technical accuracy.